### PR TITLE
Added docs for Kpi rows improvements

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: src/content/1.7/modules/concepts/controllers/kpi-blocks.md [hunk 1]
```diff
@@ -57,3 +57,24 @@ You can follow these steps to easily add a KPI row to a modern page:
...
     ```
+
+## Alter an existing Kpi row
+{{< minver v="1.7.6" title="true" >}}
+
+A hook allows you to alter the list of an existing Kpi row of the Back Office.
+
+This hook is dynamic and is dispatched after the *Kpi row identifier*.
+
+For instance, for a Kpi row identified by "foo":
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/controllers/kpi-blocks.md]: For, for it's redundant.
Maybe?
```suggestion
For instance, with a Kpi row identified by "foo":
```

File: src/content/1.7/modules/concepts/controllers/kpi-blocks.md [hunk 2]
```diff
@@ -57,3 +57,24 @@ You can follow these steps to easily add a KPI row to a modern page:
...
+
+```php
+// we are in a module
+public function hookActionFooKpiRowModifier(array $params)
+{
+    dump($params['kpis']); // access the complete list
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/controllers/kpi-blocks.md]: add a comment with a link to https://symfony.com/doc/current/components/var_dumper.html#the-dump-function it's not a native function and it's specific to a symfony component

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 3]
```diff
@@ -2101,4 +2101,20 @@ validateCustomerFormFields
 : 
     Located in: /classes/form/CustomerForm.php
 
+action<KpiIdentifier>KpiRowModifier
+: 
+    Available since: {{< minver v="1.7.6" >}}
+
+    This hook allow to alter the list of Kpis used in a Kpi row.
+This hook is called just before the validation and the  building of the KpiRow.
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md]:
```suggestion
    This hook is called just before the validation and the  building of the KpiRow.
```

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 4]
```diff
@@ -2101,4 +2101,20 @@ validateCustomerFormFields
 : 
     Located in: /classes/form/CustomerForm.php
 
+action<KpiIdentifier>KpiRowModifier
+: 
+    Available since: {{< minver v="1.7.6" >}}
+
+    This hook allow to alter the list of Kpis used in a Kpi row.
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md]: ?
```suggestion
    This hook allows to alter the list of Kpis used in a Kpi row.
```

File: src/content/1.7/modules/concepts/hooks/list-of-hooks.md [hunk 5]
```diff
@@ -2101,4 +2101,20 @@ validateCustomerFormFields
...
+This hook is called just before the validation and the  building of the KpiRow.
+
+    Located in: /controllers/front/listing/CategoryController.php
+
+    Parameters:
+    ```php
+    array(
+        'kpis' => KpiInterface[] $kpis
+    ),
+    `
```
Comment (PierreRambaud) [inline src/content/1.7/modules/concepts/hooks/list-of-hooks.md]: You didn't close the line 2114 =>  ```

File: src/content/1.7/modules/concepts/controllers/kpi-blocks.md (hunk 2 above)
Comment (mickaelandrieu) [reply src/content/1.7/modules/concepts/controllers/kpi-blocks.md]: I prefer relying on ``var_dump()`` function then

**Comment by mickaelandrieu** [comment]
All comments addressed: thanks for the review @PierreRambaud !

**Pull request merged**